tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -86,3 +98,154 @@ Verify Service Provides "Get Started" Button In The Explore Page
   Wait Until Page Contains Element    ${ODH_DASHBOARD_SIDEBAR_HEADER_TITLE}   timeout=10   error=${app_name} does not have sidebar with information in the Explore page of ODS Dashboard
   Page Should Contain Element    ${ODH_DASHBOARD_SIDEBAR_HEADER_GET_STARTED_ELEMENT}   message=${app_name} does not have a "Get started" button in ODS Dashboard
 
+Check HTTP Status Code
+    [Arguments]  ${link_to_check}  ${expected}=200
+    ${response}=    RequestsLibrary.GET  ${link_to_check}   expected_status=any
+    Run Keyword And Continue On Failure  Status Should Be  ${expected}
+    [Return]  ${response.status_code}
+
+Load App Info JSON File
+    ${APPS_DICT}=  Load Json File  ${APPS_DICT_PATH}
+    ${APPS_DICT}=  Set Variable  ${APPS_DICT}[apps]
+    Set Suite Variable  ${APPS_DICT}
```

Comment:
This should return the variable rather than setting it as a global suite variable

Reply:
it it done in this way to be used in the Suite Setup.